docs/app/Examples/collections/Breadcrumb/Content/BreadcrumbIconDividerPropExample.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import { Breadcrumb } from 'stardust'
+
+export default class BreadcrumbIconDividerPropExample extends Component {
+
+  sections = [
+    { text: 'Home', link: true },
+    { text: 'Registration', link: true },
+    { text: 'Personal Information', active: true },
+  ];
+
+  render() {
+    return (
+      <Breadcrumb icon='right-angle' sections={this.sections} />
```

Comment:
While we're at the refactors, we might as well drop React.Component and use stateless functional examples as well:

```js
import React from 'react'
import { Breadcrumb } from 'stardust'

const sections = [...]

const BreadcrumbDividerPropExample = () => (
  <Breadcrumb divider='/' sections={sections} />
)

export default BreadcrumbDividerPropExample
````